packages/@webex/plugin-meetings/test/unit/spec/roap/turnDiscovery.ts

```diff
@@ -220,6 +229,40 @@ describe('TurnDiscovery', () => {
       checkFailureMetricsSent();
     });
 
+    it('resolves with undefined when udp is reachable', async () => {
```

Comment:
I think it would be worth to modify one of existing tests that do turn discovery to have some reachability results in localstorage (all failed) so that we can be sure that turn discovery is done when all results have udp and tcp reachable as "false"